src/components/settings/SettingsSSOTab/ExistingSSOConfigs.jsx

```diff
@@ -81,107 +81,105 @@ const ExistingSSOConfigs = ({
   };
 
   return (
-    <>
-      <span>
-        <ConfigError
-          isOpen={errorIsOpen}
-          close={closeError}
-          configTextOverride={errorModalText}
-        />
-        <p>
-          Enabling single sign-on for your edX account allows quick access to
-          your organization’s learning catalog
-        </p>
-        <h4 className="mt-5 mb-4">Existing configurations</h4>
-        <CardGrid
-          className="mb-3 mr-3"
-          columnSizes={{
-            xs: 6,
-            s: 5,
-            m: 4,
-            l: 4,
-            xl: 3,
-          }}
-        >
-          {configs.map((config) => (
-            <Card
-              className="pb-4"
-              key={config.name + config.id}
-            >
-              <Card.Header
-                title={(
-                  <div className="justify-content-center">
-                    {config.display_name}
-                  </div>
+    <span>
```

Comment:
nit: unrelated to your change, but we're using block elements (`p`) inside of inline elements (`span`) which is typically an anti-pattern / not best practice. Any reason not to change the `span` to be `div` here?